net/redisclient.go

```diff
@@ -226,11 +248,67 @@ func NewRedisRingClient(ro *RedisOptions) *RedisRingClient {
 		r.ring = redis.NewRing(ringOptions)
 		r.log = ro.Log
 		r.metricsPrefix = ro.MetricsPrefix
+
+		if ro.AddrUpdater != nil {
+			if ro.UpdateInterval == 0 {
+				ro.UpdateInterval = defaultUpdateInterval
+			}
+			go r.startUpdater(context.Background())
```

Comment:
`context.Background()` argument is not used

Reply:
minor: maybe it should be `runUpdater`

Reply:
fixed